models/agent_helpers.go

```diff
@@ -359,8 +359,7 @@ func CreateNodeExporter(q *reform.Querier, pmmAgentID string, customLabels map[s
 		return nil, err
 	}
 	if !isPushMetricsSupported(pmmAgent.Version) {
-		return nil, errors.Errorf("cannot use push_metrics_enabled with pmm_agent version=%q,"+
-			" it doesn't support it, minimum supported version=%q", pointer.GetString(pmmAgent.Version), PMMAgentWithPushMetricsSupport.String())
+		pushMetrics = false
```

Comment:
How does user know whats wrong when no error is returned/printed? Or does it fallback to pull mode automatically?

Reply:
@jprukner It will continue with pull mode.

Reply:
It should not continue with pull mode if push mode was requested explicitly.

That's not the right place to silently ignore passed parameter: that's a wrong layer, and that breaks Inventory API that should do what it is told to do, not try to be smart and silently "fix" things. Instead, the caller should make sure that correct parameter is passed.

Reply:
Moved.